app/views/mortgage_calculator/stamp_duties/_form_step1.html.erb

```diff
@@ -1,6 +1,9 @@
 <div class="mortgagecalc__panel mortgagecalc__panel--full">
   <div class="intro"><%= I18n.t('stamp_duty.title') %></div>
   <p class="mortgagecalc__intro"><%= I18n.t('stamp_duty.subtitle') %></p>
+  <p class="mortgagecalc__intro">
+     <%= I18n.t('stamp_duty.second_subtitle', :href=>link_to( t('stamp_duty.href_second_subtitle'), t('stamp_duty.url_second_subtitle') ) ).html_safe %>
```

Comment:
Could we do:

```erb
<%= I18n.t('stamp_duty.second_subtitle') %>
<%= link_to t('stamp_duty.href_second_subtitle'), t('stamp_duty.url_second_subtitle') %>
```

That'd save us an `html_safe`. Maybe there's a whitespace concern I'm not aware of tho.